src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/WorkbookRepository.kt

```diff
@@ -245,12 +241,19 @@ class WorkbookRepository(private val db: IDatabaseAccessors) : IWorkbookReposito
             .subscribe {
                 content.selectedTake = it.value?.let { wbTake -> takeMap[wbTake] }
                 db.updateContent(content)
+                    .subscribe()
             }
 
         /** Initial Takes read from the DB. */
         val takesFromDb = db.getTakeByContent(content)
             .flattenAsObservable { list: List<ModelTake> -> list.sortedBy { it.number } }
-            .map { workbookTake(it) to it }
+            .map { modelTake ->
+                val wbTake = when (modelTake) {
+                    content.selectedTake -> initialSelectedTake
+                    else -> workbookTake(modelTake)
```

Comment:
This fixes a bug where we were creating two different workbook takes for the selected take